### PR TITLE
[PVR] small tweaks

### PR DESCRIPTION
**Pull request by Glenn-1990**
fixes:
1) current position was off by a few % 
2) playing tag was updated with old timeshift info 

@ksooo

### CODE REVIEW

File: xbmc/epg/EpgInfoTag.cpp [hunk 1]
```diff
@@ -263,9 +263,9 @@ float CEpgInfoTag::ProgressPercentage(void) const
   iDuration = endTime - startTime > 0 ? endTime - startTime : 3600;
 
   if (currentTime >= startTime && currentTime <= endTime)
-    fReturn = ((float) currentTime - startTime) / iDuration * 100;
```
Comment (Glenn-1990) [inline xbmc/epg/EpgInfoTag.cpp]: A 32-bit float can only represent integers till 24 bit long (without rounding), time_t of course exceeds this and this results in a miscalculation. So doing the time_t subtraction as int (and cast to float afterwards) will solve this.

**Comment by Jalle19** [comment]
If it works, +1

**Comment by Glenn-1990** [comment]
@ksooo good to go?

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
Thx, @Glenn-1990

**Pull request merged**